doc/manual/installation.md

```diff
@@ -203,6 +228,20 @@ If you are using a local MySQL server the database configuration should look lik
     # database can hold 4-byte UTF-8 characters like emoji.
     #DATABASE_ENCODING=utf8mb4
 
+If you are using a local PostgreSQL server the database configuration should look like this (use the password of the huginn PostgreSQL user you created earlier):
+
+    DATABASE_ADAPTER=postgresql
+    DATABASE_RECONNECT=true
+    DATABASE_NAME=huginn_production
+    DATABASE_POOL=20
+    DATABASE_USERNAME=huginn
+    DATABASE_PASSWORD='$password'
+    DATABASE_HOST=localhost
+    DATABASE_PORT=5432
+    DATABASE_SOCKET=/var/run/postgresql/.s.PGSQL.5432
```

Comment:
Nit: I *think* this is redundant with `DATABASE_HOST`, I don't know which one takes precedence by default, but I am pretty sure using either of the two options would be enough.

Reply:
I don't know about the precedence, I just tried to keep the PostgreSQL stanza in line with the MySQL/MariaDB example which was already present.